airbyte-config/config-models/src/main/resources/types/SyncStats.yaml

```diff
@@ -7,13 +7,17 @@ type: object
 required:
   - recordsEmitted
   - bytesEmitted
-additionalProperties: false
+additionalProperties: true
 properties:
   recordsEmitted:
     type: integer
   bytesEmitted:
     type: integer
-  stateMessagesEmitted: # TODO make required once per-stream state messages are supported in V2
+  sourceStateMessagesEmitted: # TODO make required once per-stream state messages are supported in V2
```

Comment:
I changed the name of this property because this was counting the state messages emitted from a source connector, not the total number of state messages emitted. I updated additionalProperties to true because I *think* that is needed to ensure that we don't have issues with older data that have the `stateMessagesEmitted` property, but let me know if I'm mistaken. cc: @benmoriceau since I think you might have experience with this

Reply:
👍 on the name change - please also update the PRD and tech spec to match